README.md

```diff
@@ -10,30 +10,22 @@ status: draft
 
 # Introduction
 
-In this tutorial, we are going to build a simple web application using [AdonisJS](https://adonisjs.com/) and integrate it with [Materialize](https://materialize.com).
+In this tutorial, we are going to build a web application using [AdonisJS](https://adonisjs.com/) and integrate it with [Materialize](https://materialize.com) to .
 
-# Prerequisits
+# Prerequisites
 
-You would need to have the following things installed before you begin:
+You need to have the following things installed before getting started:
 
-* [Install Docker and Docker Compose](https://www.digitalocean.com/community/tutorials/how-to-install-and-use-docker-compose-on-ubuntu-20-04)
-* [Install Node.js](https://devdojo.com/ruanbekker/install-nodejs-on-linux-using-nvm)
+* [Docker and Docker Compose](https://www.digitalocean.com/community/tutorials/how-to-install-and-use-docker-compose-on-ubuntu-20-04)
+* [Node.js](https://devdojo.com/ruanbekker/install-nodejs-on-linux-using-nvm)
 
 # What is Materialize
 
-**[Materialize](https://materialize.com) is a streaming database** that takes sources of data coming from sources like Kafka, PostgreSQL, S3 buckets, and more and allows you to create views that aggregate data on your event stream.
+**[Materialize](https://materialize.com) is a streaming database** that takes data coming from sources like Kafka, PostgreSQL, S3 buckets, and more and allows you to effectively transform it in real-time using SQL.
 
-The magic is that the views are translated under the hood to dataflows and it allows Materialize to maintain the views incrementally in real-time. 
+Unlike a traditional database, Materialize is able to incrementally maintain views on top of streaming data, providing fresh and correct results as new data arrives. This means that, instead of recomputing the view from scratch every time it needs to be updated, it only does work proportional to the changes in its inputs, so it is fast and efficient.
 
-A normal view would do a full scan of the data every time it needs to be updated, whereas Materialize only does the work to maintain the view based on events that come in, so it is much faster and efficient.
-
-To put this into a perspective, here is an example `SELECT` statement execute on the same data both from Materialize and PostgreSQL:
```

Comment:
What were you trying to show with these pictures? I think something similar could make sense if you e.g. showed data changing in the view that is powered by Materialize, otherwise I don't really see the value of having this.